aws/scenarios/fakeintake/run.go

```diff
@@ -0,0 +1,29 @@
+package fakeintake
+
+import (
```

Comment:
Note: This file doesn't seem to belong to this PR

Reply:
ah good catch, thank you !